guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -42,6 +41,10 @@ By default, {ProjectServer} uses the language and timezone settings of the user
 +
 The `--auth-source-id 1` setting means that the user is authenticated internally, you can specify an external authentication source as an alternative.
 Add the `--admin` option to grant administrator privileges to the user.
-Specifying organization IDs is not required, you can modify the user details later using the `update` subcommand.
+Specifying organization IDs is not required.
++
+You can modify the user details later by using the `update` subcommand.
```

Comment:
I am unsure about the wording. In here, I lean towards `hammer user update` because at first, it's unclear what "subcommand" means/refers to.

Reply:
+1

Reply:
Applied.